platform/pulpcore/app/serializers/status.py

```diff
@@ -0,0 +1,27 @@
+from gettext import gettext as _
+from rest_framework import serializers
+
+from pulpcore.app.serializers.task import WorkerSerializer
+
+
+class StatusSerializer(serializers.Serializer):
+    """
+    Serializer for the status information of the app
```

Comment:
Can we be more specific for each of the fields about what it will return? Maybe the ``WorkerSerializer`` will take care of itself, since that's a known data structure that DRF may be able to auto-document as-is. That would be good to verify though.

The rest would benefit from more detail, so an integrator doesn't need to just try to get the value, see what is returned, hope they interpret it correctly, and hope it stays consistent. Ideally someone should be able to write tests based on the API docs, which I assume comes from this class.